### PR TITLE
Update rubygem-foreman_puppet to 1.0.0

### PR DESCRIPTION
**Pull request by ezr-ondrej**

### CODE REVIEW

**Review by ehelms — APPROVED** [review]

File: packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec [hunk 1]
```diff
@@ -8,7 +8,7 @@
 %global foreman_min_version 2.4
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.0.0.rc.2
+Version: 1.0.0
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec:11]: @ezr-ondrej @ehelms this version is smaller than 1.0.0.rc2 if you ask RPM...

Comment (ehelms) [reply packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec:11]: Oh cause of the whole we publish all the versions of plugins thing? Fair. @ezr-ondrej the easiest solution is release a 1.0.1 :D

Comment (evgeni) [reply packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec:11]: Yeah. And because we didn't catch that when rc1 want out.

Comment (ezr-ondrej) [reply packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec:11]: might that be an issue?

**Pull request merged**